### PR TITLE
 Implemented issue #253: Duration.toISO() doesn't include milliseconds

### PR DESCRIPTION
**Pull request by bogdanpolak**
@icambron 
#253

### CODE REVIEW

**Comment by bogdanpolak** [comment]
Sorry for all this commits. I started to fix issue on my fork. Now I'm not sure how to remove this old commits them. Maybe now I should make new fork of luxon/master? Or maybe can just accept all this stuff?

**Comment by icambron** [comment]
I'll squash when I merge, so don't worry about the commits

File: src/duration.js [hunk 1]
```diff
@@ -381,6 +381,7 @@ export default class Duration {
    * @example Duration.fromObject({ months: 4, seconds: 45 }).toISO() //=> 'P4MT45S'
    * @example Duration.fromObject({ months: 5 }).toISO() //=> 'P5M'
    * @example Duration.fromObject({ minutes: 5 }).toISO() //=> 'PT5M'
+   * @example Duration.fromObject({ milliseconds: 6 }).toISO() //=> 'PT0.011S'
```
Comment (icambron) [inline src/duration.js]: Shouldn't 6 milliseconds be `PT0.006S`?

Comment (bogdanpolak) [reply src/duration.js]: of course :-)

**Comment by icambron** [comment]
Looks good, thanks!

**Comment by icambron** [comment]
(I'm going to do a new release soon)

**Pull request merged**